### ISSUE TITLE
[Other - Accessibility] Almost-match between visible text and screen reader only text. (00.00.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43976]
#### VFS team name
Debt Resolution

#### VFS product name
Combined VA Debt Portal

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
When viewing details on a copay, the link to view the statement includes visible text that has been hidden from screen reader users, and screen reader only text that has been hidden visibly. This also happens when viewing the statement itself with the link to download a PDF copy of the statement.

That technique should be used when the visible text alone might be ambiguous to screen reader users, or when some extra information would make the link more usable (eg. sighted users see an icon that needs to be communicated with some alternative text). In this case, the two versions of the text are almost the same and the slight difference doesn't seem to add much value.

One unintended consequence: For voice command users who speak a command like `Click [link text]`, they'll likely say the text they see on screen --- but their software will attempt to match against the hidden text instead. Since the phrasing of the two versions are a little different, they might not get a match.

### Link, screenshot or steps to recreate
Logged in as test user 80, navigate to the [June 5 statement](https://staging.va.gov/manage-debt-and-bills/summary/copay-balances/702c94e3-be4a-44f5-be80-f261d4baddfd/detail/statement). The download link is coded as:

```
<a download="..." href="...">
     <span aria-hidden="true">
          [visible link]
     </span>
     <span class="sr-only">
          [text announced to screen reader]
     </span>
</a>
```

The visible link text is `Download your [date] statement (PDF)`, and the screen reader version is `Download [date] dated medical copay statement (PDF)`

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
I don't think there's any reason to provide screen reader users a different version of this link text. A `<span aria-hidden="true">` around the download icon `i` element is appropriate, but the link text should be outside the `aria-hidden` span and the `sr-only` span should be removed.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
